site/en/codelabs/openthread-network-simulator/index.lab.md

```diff
@@ -418,78 +417,61 @@ OTNS enables users to move nodes easily through `OTNS-CLI` or `OTNS-Web`.
 
 ### Move node through `OTNS-CLI`
 
-Move node 5 to a new location:
+Move the Border Router node 9 to a new location:
 
 ```console
-> move 5 600 300
+> move 9 50 50
 Done
 ```
 
-Since now node 5 is far from the other Router, they should lose connectivity to
-each other, and after about 12s (120s in simulating time) both become Leaders of
-their own partition:
-
-<img src="img/c06b4d0a4f183299.png" alt="c06b4d0a4f183299.png" width="679.11" />
-
-> aside positive
->
-> **Tip:** Nodes have limited radio transmission range in OTNS simulation.
-
 ### Move node through OTNS-Web
 
-Move node 5 back to the original location by dragging. The two partitions should
-merge back into one partition:
+Move node 5 all the way to the bottom right, by dragging. Because node 5 is now out of radio coverage of 
+the other Routers, it forms its own partition with a new Partition ID. The Partition IDs can be checked 
+on the node information panel by clicking the nodes.
 
-<img src="img/9ba305c4c5a5f892.png" alt="9ba305c4c5a5f892.png" width="673.06" />
+<img src="img/10_9n_move_router_away.png" alt="Node 5 is moved away from the other nodes and forms a new partition" width="720" />
 
+Note that a single green line is still drawn between node 5 and node 9. This is often due to stale information about a 
+child, which is still kept in the child table of a former parent. Or it could be stale information about the former 
+router-to-router link between node 9 and node 5. (Or possibly, in this case, even a rendering bug.) 
+Eventually, stale information is cleaned up on the nodes after the appropriate timeout.
 
 ## Delete Nodes
 Duration: 01:00
 
 
 ### Delete nodes through `OTNS-CLI`
 
-Delete node 8:
+Delete node 5:
 
 ```console
-> del 8
+> del 5
 Done
 ```
 
-Node 8 should disappear from the simulation:
+Node 5 should disappear from the simulation:
 
-<img src="img/18156770d9f8bf83.png" alt="18156770d9f8bf83.png" width="624.00" />
+<img src="img/11_8n_del_router.png" alt="Node 5 is deleted from the simulation" width="720" />
 
 ### Delete nodes through `OTNS-Web`
 
-Select node 5 and click the
-<img src="img/7ff6afd565f4eafc.png" alt="7ff6afd565f4eafc.png" width="55.25" />
-button on the `Action Bar` to delete node 5:
+Select the Border Router node 9 and click the
+<img src="img/7ff6afd565f4eafc.png" alt="Delete button" width="55.25" />
+button on the `Action Bar` to delete node 9:
 
-<img src="img/d4079cceea0105f0.png" alt="d4079cceea0105f0.png" width="642.12" />
+<img src="img/12_7n_del_br.png" alt="Border Router node 9 is deleted" width="642.12" />
```

Comment:
I would make the width here consistent with the others, otherwise they are all fine